### ISSUE TITLE
eInvoice A2A integration

### ISSUE THREAD

**Issue by gr4per** [OpusCapita/businessnetwork#54]
supplier selects eInvoice channel
supplier gets opuscapita terms and conditions with checkbox to accept it
supplier next gets dekabank terms and conditions with checkbox to accept it

once supplier finishes the flow, email to service now will be send to trigger manual implementation

implementation team needs to setup the connection, then feedback status update to BNP via API call

Please attach:
email template (english / german)
email receiver address + subject
opus terms and condition (english / german)
deka terms and conditions (english / german)

**Comment by pullihe1** [comment]
1. Process step on PPT:
[Service_Configuration_einvoice.pptx](https://github.com/OpusCapita/businessnetwork/files/1288520/Service_Configuration_einvoice.pptx)

2. Opus terms and conditions:
[Business_Network_Portal_Terms of Use_20170904.pdf](https://github.com/OpusCapita/businessnetwork/files/1288524/Business_Network_Portal_Terms.of.Use_20170904.pdf)

3. DekaBank Guideline eng ( no other Dekabank specific terms&conditions):
[Supplier-Guide_E-invoicing_OC_Dekabank_final.pdf](https://github.com/OpusCapita/businessnetwork/files/1288529/Supplier-Guide_E-invoicing_OC_Dekabank_final.pdf)

4. Email template (OC-support notification email) eng.
[Einvoice_configuration_email_template_eng.docx](https://github.com/OpusCapita/businessnetwork/files/1288532/Einvoice_configuration_email_template_eng.docx)

**Comment by pullihe1** [comment]
German language translations will come hope end of next week. First release will be done in English.

**Comment by gr4per** [comment]
Heikki,

the general terms and conditions (2.) in above list is already accepted by user on registration as it is not service specific.
Do we have any specific requirements for the einvoice service ?

**Comment by pullihe1** [comment]
No, there will be not any service specific nor customer terms and conditions.

**Comment by pullihe1** [comment]
Hi updates to previously shared materials:

1. Updated email notification template
[Einvoice_configuration_email_template_final_eng.docx](https://github.com/OpusCapita/businessnetwork/files/1298400/Einvoice_configuration_email_template_final_eng.docx)

2. Updated Supplier Guide:
[Supplier-Guide_E-invoicing_OC_Dekabank_final_final.pdf](https://github.com/OpusCapita/businessnetwork/files/1298411/Supplier-Guide_E-invoicing_OC_Dekabank_final_final.pdf)

**Comment by gr4per** [comment]
ok, so the first step is
"I want to send E-invoices to DekaBank through a third party Operator"
"Ich möchte elektronische Rechnungen an DekaBank über einen anderen Anbieter schicken"

this will create InChannelConfig for supplier, type einvoice, billingModelId = external

**Comment by gr4per** [comment]
Email sending to be done via A2A integration (switch on InChannel type)
pdf = process as of now, FTP upload
einvoice = new process: send email accordign Heikkis template
